viper_test.go

```diff
@@ -1166,16 +1166,22 @@ func TestBindPFlagDetectNilFlag(t *testing.T) {
 	assert.Error(t, result)
 }
 
-func TestBindPFlagStringToString(t *testing.T) {
+func TestBindPFlagCSVKeyValue(t *testing.T) {
 	tests := []struct {
 		Expected map[string]string
 		Value    string
 	}{
+		// stringToString
 		{map[string]string{}, ""},
 		{map[string]string{"yo": "hi"}, "yo=hi"},
 		{map[string]string{"yo": "hi", "oh": "hi=there"}, "yo=hi,oh=hi=there"},
 		{map[string]string{"yo": ""}, "yo="},
 		{map[string]string{"yo": "", "oh": "hi=there"}, "yo=,oh=hi=there"},
+		//stringToInt
```

Comment:
```suggestion
		// stringToInt
```

Reply:
I wonder what would have happened with this test cases without adding `stringToInt` above.

Reply:
before this, int values were merely being treated as integers

Reply:
done above suggested change